src/components/icons/ArrowDownIcon/ArrowDownIcon.tsx

```diff
@@ -0,0 +1,9 @@
+import * as React from 'react';
+import Icon, { IconProps } from '../Icon';
+import SVGArrowDownIcon from './SVGArrowDownIcon';
+
+const ArrowDownIcon: React.FC<IconProps> = (props1) => (
```

Comment:
почему `props1` а не просто `props`?)
```suggestion
const ArrowDownIcon: React.FC<IconProps> = (props) => (
```

Reply:
поправил